packages/redux-simple-auth/src/authorizers/bearerToken.js

```diff
@@ -0,0 +1,5 @@
+export default ({ token: accessToken }, header) => {
```

Comment:
I'm ok calling this `token` here. No need to rename the field. I did it with the other authorizer mostly to convert from camel_case to snakeCase which isn't needed here.